OpenSim/Actuators/Test/testMuscles.cpp

```diff
@@ -638,51 +512,23 @@ void testRigidTendonMuscle()
     double act0 = 0.5;
     Constant control(act0);
 
-    Sine sineWave(0.1, SimTK::Pi, 0);
-
-    /*
-    void simulateMuscle(const Muscle &aMuscle, 
-                    double startX, 
-                    double act0, 
-                    const Function *motion, 
-                    const Function *control, 
-                    double integrationAccuracy,
-                    int testType,
-                    double testTolerance,
-                    bool printResults);
-    */
+    Sine motion(OptimalFiberLength0, 2.0*SimTK::Pi, 0);
 
     // concentric
     simulateMuscle(muscle, 
         x0, 
         act0, 
         NULL, 
         &control, 
-        IntegrationAccuracy,
-        SimulationTest,
-        SimulationTestTolerance,
-        false);
-    // eccentric 
-    simulateMuscle(muscle, 
-        x0, 
-        act0, 
-        &sineWave, 
-        &control, 
-        IntegrationAccuracy,
-        CorrectnessTest,
-        CorrectnessTestTolerance,
-        false);
+        true);
 
+    // eccentric 
     simulateMuscle(muscle, 
         x0, 
         act0, 
-        &sineWave, 
+        &motion,
         &control, 
-        IntegrationAccuracy,
-        InitializationTest,
-        InitializationTestTolerance,
         false);
-
 }
```

Comment:
Not exactly. The same simulation was executed twice, but one evaluated the "CorrectnessTest" and the other did the "InitializationTest" (which was just a stub).  I got rid of these *flags* so that all conditions are tested.